bbc/simorgh
# WSTEAM1-664 - Next.js `.app` route support

Resolves JIRA https://jira.dev.bbc.co.uk/browse/WSTEAM1-664

Overall changes
======
- Adds check for the page URL to set the `isApp` flag for `.app` routes

Code changes
======
- Updates server side request to check if the route is `.app` and sets the `isApp` flag as appropriate
- Adds the `<meta name="robots" content="noindex" />` meta tag when under `.app` route

Testing
======
1. Visit http://localhost:7081/pidgin/live/c7p765ynk9qt locally
2. Confirm the header and footer are shown
3. View 'page source' and confirm the `content="noindex"` meta tag is not present
4. Visit http://localhost:7081/pidgin/live/c7p765ynk9qt.app
5. Confirm the header and footer are removed
6. View 'page source' and confirm the `content="noindex"` meta tag is present

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
